tests/integration/ovs_test.py

```diff
@@ -44,7 +44,7 @@
 from .testlib.vlan import vlan_interface
 
 
-BOND1 = "bond1"
+BOND0 = "bond0"
```

Comment:
This is unlated changed.

Reply:
I just did it because I found most of the tests are using "0" as first interface identifier, e.g team0, br0, etc..

I will remove it, it is just a nit. Thanks

Reply:
Done!